### DISCUSSION TITLE
Minimum username length should be 1 or configurable, and have less restrictions

### DISCUSSION THREAD

**Discussion by 4-9** [pelican-dev/panel#744] [General]
As someone who likes to use a two character username, the recent inclusion of a 3 character minimum (like Pterodactyl) is a mild inconvenience.
If there isn't any objective reason for this minimum could this either default to 1, or be configurable somewhere?

Second, and somewhat a duplicate of other discussions.
Please allow usernames to contain alphanumeric characters in any order

**Comment by rmartinoscar** [comment]
It's the industry standard minimum, modern platforms like GitHub, Twitter, and Instagram require usernames to be at least 3 characters long.
> Please allow usernames to contain alphanumeric characters in any order
Its already the case alphanumeric means a-z0-9 (It also includes A-Z but [we made username case insensitive](https://github.com/pelican-dev/panel/pull/714/files#diff-37a2d7d879a7de3f7d73b2975cd22be9f41ec10c334a0dda9ad1e869332e4ecbR260))

**Reply by 4-9** [reply]
I disagree that it's a standard, I can go quote some other platforms like Discord, and Steam which don't restrict usernames in such ways.

It would be great if I could just use my normal username "49" without an arbitrary minimum name length imposed on me for no real justification.
